src/DateRange/DateRange.tsx

```diff
@@ -43,15 +43,19 @@ export const DateRange: React.FC<Props> = ({
 	addFilterButtonCaption,
 	cancelFilterButtonCaption
 }) => {
-	const [dateFrom, setDateFrom] = React.useState<Date>(getWeekBeforeNow());
-	const [dateTo, setDateTo] = React.useState<Date>(getNow());
-	const [timeFrom, setTimeFrom] = React.useState<Time>(
-		getMinutes(getWeekBeforeNow())
-	);
-	const [timeTo, setTimeTo] = React.useState<Time>(getMinutes(getNow()));
+	const weekBeforeNow = getWeekBeforeNow();
+	const dateNow = getNow();
+	const minutesFrom = getMinutes(weekBeforeNow);
+	const minutesTo = getMinutes(dateNow);
+
+	const [dateFrom, setDateFrom] = React.useState<Date>(weekBeforeNow);
+	const [dateTo, setDateTo] = React.useState<Date>(dateNow);
+	const [timeFrom, setTimeFrom] = React.useState<Time>(minutesFrom);
+	const [timeTo, setTimeTo] = React.useState<Time>(minutesTo);
 	const [value, setValue] = React.useState<DateRangeValueTypes | LastPeriods>(
 		DateRangeValueTypes.NoFilter
 	);
+	const [hasError, setError] = React.useState<boolean>(false);
```

Comment:
Принято повторять название стейта в имени функции-сеттера
```suggestion
	const [hasError, setHasError] = React.useState<boolean>(false);
```